cells.py

```diff
@@ -290,11 +296,12 @@ def help(self):
                 if event.type == pygame.QUIT:
                     self.running = False
                 # and this one is for the "ESC" key
-                if event.type == pygame.KEYDOWN and event.key == pygame.K_ESCAPE:
-                        helping = False
+                if (event.type == pygame.KEYDOWN
+                    and event.key == pygame.K_ESCAPE):
+                    helping = False
```

Comment:
The indentation here should make it obvious where the condition ends and that's difficult to tell at first glance.

Reply:
Yep and a flake8 was present, fixed